output.R

```diff
@@ -28,6 +28,8 @@ helpText <- "
 #'
 #'    --help, -h:      show this help text and exit
 #'    --test, -t:      tests output.R without actually starting any run
+#'    --renv=<path>    load the renv located at <path>, incompatible with --update
```

Comment:
These two lines seem fine for me, I did not check the rest. Note that Michajas magic autocomplete scripts will not work on these, as this requires a `,` behind the `--update`. You could also add `-u` as shortcut for `--update` if you like by adding `u = "--update"` to the `.flags` argument of `readArgs`. But not that important, I think.

Reply:
> Note that Michajas magic autocomplete scripts will not work on these, as this requires a `,` behind the `--update`.

That's your doing, not mine.  They are "my" "magic" autocomplete scripts no more.